### PR TITLE
[Adjustment][Merv] Add linting rules and Authlayout app shell

### PR DESCRIPTION
**Pull request by jdesolate**
- Added import sort for linting rules
- Applied linter on files
- Added app shell on auth layout

### CODE REVIEW

File: pages/404.tsx [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
+    fontWeight: 900,
 
-  description: {
-    maxWidth: 500,
-    margin: 'auto',
-    marginTop: theme.spacing.xl,
-    marginBottom: theme.spacing.xl * 1.5,
+    textAlign: 'center',
   },
 }));
```
Comment (tapsTapayan) [inline pages/404.tsx]: Styles must be on the stylesheet

**Review by tapsTapayan — CHANGES_REQUESTED** [review]
Resolve comments lang @jdesolate

File: pages/404.tsx (hunk 1 above)
Comment (jdesolate) [reply pages/404.tsx]: resolved

**Review by tapsTapayan — APPROVED** [review]
approved @jdesolate

**Pull request merged**